includes/class-wc-gateway-stripe.php

```diff
@@ -673,12 +673,14 @@ public function redirect_to_stripe_checkout( $order ) {
 	/**
 	 * Creates a new WC_Stripe_Customer if the visitor chooses to.
 	 *
-	 * @since 4.2.0
-	 * @param WC_Order $order The order that is being created.
+	 * @since 4.2.0 Isolated as a separate method.
+	 * @since 4.2.0 Added `force_save_source` to ensure subscriptions work.
```

Comment:
No need to put 2 `@since` annotations for the same version (`4.2.0`), because no code could possibly rely on this function without the `force_save_source` param